### DISCUSSION TITLE
5 days and still no academic benefits

### DISCUSSION THREAD

**Discussion by Sonjoaquin** [community/community#134930] [GitHub Education] [answered]
### Hi everyone,

Hello, good morning.
I have had my application approved since August 2 and I still cannot enjoy any of the benefits.
![image](https://github.com/user-attachments/assets/9293f0dd-6974-4d7b-9d23-14acb7f1b6f4)
When I try to use my github account to log in when accessing any of the offers, I get a message saying that it could not verify that I am a student and that the process may take a few days.
How much longer do I have to wait?
I should already have access to the academic benefits.
![image](https://github.com/user-attachments/assets/f62b6668-ac90-4087-a169-316a8dda6d6f)

**Comment by davevad93** [comment, marked as answer]
Hi @Sonjoaquin ,

That's odd, you should indeed have access to your academic benefits. Try contacting education support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your issue.

**Comment by Chaitanya029** [comment]
Hi @Sonjoaquin 

Sorry to hear that you're experiencing issues with accessing your academic benefits despite having your application approved.

The error message you're seeing suggests that there might be a delay in verifying your student status. This can sometimes take a few days, but, understandably, you're eager to access the benefits.

Here are a few potential steps you could take:

Check your email: Make sure you haven't missed any emails from GitHub regarding your application or student status verification.
Verify your student status: Double-check that your student status is correctly linked to your GitHub account. You can do this by going to your GitHub account settings > Applications > Student Developer Pack.
Contact GitHub Support: Reach out to GitHub Support directly and provide them with your application ID and a screenshot of the error message. They should be able to help you resolve the issue or provide more information on the status of your application.
If you've already waited for a few days, it's likely that there's an issue that needs to be manually resolved by the GitHub team.

You can contact GitHub Support by submitting a request through the GitHub Support Portal. Make sure to include as much detail as possible about your issue, including your application ID and any relevant screenshots.
